rust/src/rfb/rfb.rs

```diff
@@ -231,7 +248,11 @@ impl RFBState {
                         consumed += current.len() - rem.len();
                         current = rem;
 
-                            self.state = parser::RFBGlobalState::TCSecurityResult;
```

Comment:
This line seems oof in this commit...

Reply:
But with a different indentation ;-)

Reply:
it's due to rustfmt , earlier the file did not seem to be formatted with it.